spanner/test/cloudexecutor/worker_proxy.go

```diff
@@ -28,6 +28,10 @@ import (
 
 	"cloud.google.com/go/spanner/executor/apiv1/executorpb"
 	"cloud.google.com/go/spanner/test/cloudexecutor/executor"
+	texporter "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/propagation"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
```

Comment:
I am worried about this dep since we are adding a direct dependency of OTEL SDK package to the main package. This will download SDK dep to customer.

Can you check if we can have a seperate dependency file for executor which now relies on main package dependency. I have done something similar for opentelemetry tests since it also had SDK dependency.

https://github.com/googleapis/google-cloud-go/tree/main/spanner/test/opentelemetry/test